src/SurveyService.Validation/Group/CreateGroupRequestValidator.cs

```diff
@@ -0,0 +1,31 @@
+using FluentValidation;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Group;
+using LT.DigitalOffice.SurveyService.Validation.Group.Interfaces;
+using LT.DigitalOffice.SurveyService.Validation.Question.Interfaces;
+
+namespace LT.DigitalOffice.SurveyService.Validation.Group;
+
+public class CreateGroupRequestValidator : AbstractValidator<CreateGroupRequest>, ICreateGroupRequestValidator
+{
+  public CreateGroupRequestValidator(
+    ICreateQuestionRequestValidator createQuestionRequestValidator)
+  {
+    RuleFor(group => group.Subject)
+      .Cascade(CascadeMode.Stop)
+      .NotEmpty()
+      .WithMessage("Subject can't be empty")
+      .MaximumLength(150)
+      .WithMessage("Subject should not exceed maximum length of 150 symbols.");
+
+    RuleFor(group => group.Description)
```

Comment:
exception if description is null

Reply:
For groups we allow empty description.
 Is it what you were asking about?

Reply:
`When (group => !string.IsNullOrEmpty(group.Description), () =>
{ 
RuleFor( ...)
}`